<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Widget</class>
 <widget class="QWidget" name="Main_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>中国移动Ping测平台</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/images/title_logo.png</normaloff>:/images/title_logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 94, 149);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1911</width>
     <height>1071</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="headerLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="logo_Lable_3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>75</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/images/top_logo.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="webBtn_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>370</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>370</width>
          <height>75</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/images/10.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pingBtn_3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>370</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>370</width>
          <height>75</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/images/2.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="bottomLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="pingmainLayout">
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QPushButton" name="unit1Btn">
            <property name="minimumSize">
             <size>
              <width>340</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>340</width>
              <height>70</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(:/images/c10.png);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="unit2Btn">
            <property name="minimumSize">
             <size>
              <width>340</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>340</width>
              <height>70</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(:/images/c2.png);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="unit3Btn">
            <property name="minimumSize">
             <size>
              <width>340</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>340</width>
              <height>70</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(:/images/c3.png);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="unit4Btn">
            <property name="minimumSize">
             <size>
              <width>340</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>340</width>
              <height>70</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(:/images/c4.png);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="PingLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QTextBrowser" name="Ping_code">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(207, 207, 207);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
